1.10/gui.md

```diff
@@ -14,36 +14,37 @@ The dashboard is the home page of the DC/OS web interface and provides an overvi
 
 ![Dashboard](/docs/1.10/img/dcos-gui.png)
 
-From the dashboard you can easily monitor the health of your cluster.
-
 *   The CPU Allocation panel provides a graph of the current percentage of available general compute units that are being used by your cluster.
 
 *   The Memory Allocation panel provides a graph of the current percentage of available memory that is being used by your cluster.
 
-*   The Task Failure Rate panel provides a graph of the current percentage of tasks that are failing in your cluster.
+*   The Disk Allocation panel provides a graph of the current percentage of available disk space that is being used by your cluster.
 
-*   The Services Health panel provides an overview of the health of your services. Each service provides a health check, run at intervals. This indicator shows the current status according to that health check. A maximum of 5 services are displayed, sorted by priority of the most unhealthy. You can click the View all Services button for detailed information and a complete list of your services.
+*   The Services Health panel provides an overview of the health of your services. Each service provides a health check, run at intervals. This indicator shows the current status according to that health check. A maximum of 5 services are displayed, sorted by priority of the most unhealthy. You can click the View All N Services button for detailed information and a complete list of your services.
 
 *   The Tasks panel provides the current number of tasks that are staged and running.
 
+*   The Component Health panel provides an overview of the health of your component. Each component provides a health check, run at intervals. This indicator shows the current status according to that health check. A maximum of 5 components are displayed, sorted by priority of the most unhealthy. You can click the View All N Components button for detailed information and a complete list of your components.
+
```

Comment:
Same comment as for enterprise -- UI elements are bold (**View All N Components**); is N here actually what the UI says, or is this a way to indicate that the UI lists the # of components? If the latter, maybe replace N with <number> to indicate it's a variable?